packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -190,13 +197,20 @@ const SceneView = ({
             false
           : gestureEnabled
       }
+      homeIndicatorHidden={homeIndicatorHidden}
+      navigationBarColor={navigationBarColor}
+      navigationBarHidden={navigationBarHidden}
       replaceAnimation={animationTypeForReplace}
       stackPresentation={presentation === 'card' ? 'push' : presentation}
       stackAnimation={animation}
       screenOrientation={orientation}
       statusBarAnimation={statusBarAnimation}
       statusBarHidden={statusBarHidden}
       statusBarStyle={statusBarStyle}
+      statusBarColor={statusBarColor}
+      statusBarTranslucent={statusBarTranslucent}
+      swipeDirection={swipeDirection}
```

Comment:
What about the logic of `swipeDirection` that sets other props correctly if not applied? (see https://github.com/software-mansion/react-native-screens/pull/1260/files#diff-d97b3df9fc6a0f0846507a3c189b483ac83a59c21db8788fc8d08449fbeb7343R176) I think it should be added here too.

Reply:
Done